src/main/java/com/cloudbees/jenkins/GitHubRepositoryNameContributor.java

```diff
@@ -30,13 +34,13 @@
      * Looks at the definition of {@link AbstractProject} and list up the related github repositories,
      * then puts them into the collection.
      */
-    public abstract void parseAssociatedNames(AbstractProject<?,?> job, Collection<GitHubRepositoryName> result);
+    public abstract void parseAssociatedNames(Job<?,?> job, Collection<GitHubRepositoryName> result);
```

Comment:
@jglick could you describe why workflow requires from plugins using Job class that is not supported by plugin internals?